seed/views/main.py

```diff
@@ -446,3 +450,159 @@ def delete_organization_inventory(request):
         'progress': 0,
         'progress_key': deleting_cache_key
     })
+
+
+def pm_integration(request):
+    return render(request, 'seed/pm_integration.html', {})
+
+
+ATTRIBUTES_TO_PROCESS = [
+    'national_median_site_energy_use',
+    'site_energy_use',
+    'source_energy_use',
+    'site_eui',
+    'source_eui'
+]
+
+
+def normalize_attribute(attribute_object):
+    u_registry = pint.UnitRegistry()
+    if '@uom' in attribute_object and '#text' in attribute_object:
+        # this is the correct expected path for unit-based attributes
+        string_value = attribute_object['#text']
+        try:
+            float_value = float(string_value)
+        except ValueError:
+            return {'status': 'error', 'message': 'Could not cast value to float: \"%s\"' % string_value}
+        original_unit_string = attribute_object['@uom']
+        if original_unit_string == u'kBtu':
+            converted_value = float_value * 3.0
+            return {'status': 'success', 'value': converted_value, 'units': str(u_registry.meter)}
+        elif original_unit_string == u'kBtu/ft²':
+            converted_value = float_value * 3.0
+            return {'status': 'success', 'value': converted_value, 'units': str(u_registry.meter)}
+        elif original_unit_string == u'Metric Tons CO2e':
+            converted_value = float_value * 3.0
+            return {'status': 'success', 'value': converted_value, 'units': str(u_registry.meter)}
+        elif original_unit_string == u'kgCO2e/ft²':
+            converted_value = float_value * 3.0
+            return {'status': 'success', 'value': converted_value, 'units': str(u_registry.meter)}
+        else:
+            return {'status': 'error', 'message': 'Unsupported units string: \"%s\"' % original_unit_string}
+
+
+@api_view(['POST'])
+def pm_integration_worker(request):
+    if 'email' not in request.data:
+        return JsonResponse('Invalid call to PM worker: missing email for PM account')
+    if 'username' not in request.data:
+        return JsonResponse('Invalid call to PM worker: missing username for PM account')
+    if 'password' not in request.data:
+        return JsonResponse('Invalid call to PM worker: missing password for PM account')
+    if 'template_name' not in request.data:
+        return JsonResponse('Invalid call to PM worker: missing template_name for PM account')
+    email = request.data['email']
+    username = request.data['username']
+    password = request.data['password']
+    template_name = request.data['template_name']
+    pm = PortfolioManagerImport(email, username, password)
+
+    possible_templates = pm.get_list_of_report_templates()
```

Comment:
This section of the code will become a separate function that will be used to populate the drop down.